### DISCUSSION TITLE
I can't start app

### DISCUSSION THREAD

**Discussion by tonyjleal** [shravan20/github-readme-quotes#177] [General]
Hello, I have forked this project, but when I executed 'npm start' throw next error:
Instead change the require of index.js in ... fetchApi.js to a dynamic import() which is available in all CommonJS modules.
 I have installed `npm i node-fetch@2.6.1` and this error has been resolved, but now show me:
 Error: ENOENT: no such file or directory, stat ... /frontend/build/index.html.

Can someone help me? what i am doing bad?

**Comment by zhravan** [comment]
This issue has been fixed, please check now
